cla_public/templates/_cookie-consent-form.html

```diff
@@ -1,97 +1,97 @@
 
 <div class="cookie-settings__form-wrapper">
-	<p class="govuk-body">We use different types of cookies. You can choose which cookies you're happy for us to use and how the data is shared.</p>
+	<p class="govuk-body">{% trans %}We use different types of cookies. You can choose which cookies you're happy for us to use and how the data is shared.{% endtrans %}</p>
 	<form method="get" data-module="cookie-settings">
     <div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
-      <h2 class="govuk-heading-l">Strictly necessary cookies</h2>
-      <p class="govuk-body">We use essential cookies to help you use the ‘Check you can get legal aid’ service. These do things such as:</p>
-      <ul class="govuk-list">
-        <li>remember your progress through our service</li>
-        <li>remember you have seen the cookies message</li>
-        <li>remember the language you have selected to view our service</li>
+      <h2 class="govuk-heading-l">{% trans %}Strictly necessary cookies{% endtrans %}</h2>
+      <p class="govuk-body">{% trans %}We use essential cookies to help you use the ‘Check you can get legal aid’ service. These do things such as:{% endtrans %}</p>
+      <ul>
+        <li>{% trans %}remember your progress through our service{% endtrans %}</li>
+        <li>{% trans %}remember you have seen the cookies message{% endtrans %}</li>
+        <li>{% trans %}remember the language you have selected to view our service{% endtrans %}</li>
       </ul>
-      <p class="govuk-body">These cookies need to be on to give you the best experience using the service.</p>
-      <p class="govuk-body"><a class="govuk-link" href="/cookies">Read more about cookies on 'Check if you can get legal aid'</a></p>
+      <p class="govuk-body">{% trans %}These cookies need to be on to give you the best experience using the service.{% endtrans %}</p>
+      <p class="govuk-body"><a class="govuk-link" href="/cookies">{% trans %}Read more about cookies on 'Check if you can get legal aid'{% endtrans %}</a></p>
     </div>
 
     <div class="govuk-form-group">
       <fieldset class="govuk-fieldset" aria-describedby="cookie-hint-1">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">Cookies used to improve 'Check you can get legal aid'</legend>
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">{% trans %}Cookies used to improve 'Check you can get legal aid'{% endtrans %}</legend>
         <span id="cookie-hint-1" class="govuk-hint">
           <p>
-            We use Google Analytics to measure how you use ‘Check you can get legal aid’ so we can improve it based on user needs. 
-            We do not allow Google to use or share the data about how you use this site. 
+            {% trans %}We use Google Analytics to measure how you use ‘Check you can get legal aid’ so we can improve it based on user needs. 
+            We do not allow Google to use or share the data about how you use this site.{% endtrans %} 
```

Comment:
Should we split this up sentence by sentence to reduce the impact of whitespace and avoid whole paragraphs being a single translation?
Like this:
`{% trans %}We use Google Analytics to measure how you use ‘Check you can get legal aid’ so we can improve it based on user needs.{% endtrans %} `
`{% trans %}We do not allow Google to use or share the data about how you use this site.{% endtrans %} `